### ISSUE TITLE
Code Contribution: [Easy] [Operator Development] elu

### ISSUE THREAD

**Issue by StrongSpoon** [FlagOpen/FlagGems#390]
## Description 任务介绍
Develop forward function for elu operator.
开发elu算子的前向功能。
## Requirements 任务要求
**Interface 接口**
`elu(Tensor self, Scalar alpha=1, Scalar scale=1, Scalar input_scale=1) -> Tensor`
**Function reference 功能参考**
https://pytorch.org/docs/stable/generated/torch.nn.functional.elu.html#torch.nn.functional.elu
**Implementation reference 实现参考**
https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/ops/relu.py

The operator should support all optional arguments defined in the interface. 
算子应支持接口中定义的所有参数选项。
Please provide both accuracy test and performance test code.
请同时提供实现正确性测试与性能测试代码。
## DDL 提交时间
Please submit a Pull Request within 1 week after accepting the assignment.
请于接取任务后一周内提交PR。

**Comment by AdvancedCompiler** [comment]
AdvancedCompiler 已认领
